README.md

```diff
@@ -1,32 +1,51 @@
-# XMTP Rust Swift helper
+# XMTP Rust Swift
 
-Do things in Rust that are hard in Swift
+Swift Package and Cocoapod that wraps an XCFramework emitted by the `bindings/xmtp_rust_swift` crate in [libxmtp](https://github.com/xmtp/libxmtp)
+
+## Usage
+
+**NOTE**: This package is NOT meant to be directly consumed by end users. Instead see [xmtp-ios](https://github.com/xmtp/xmtp-ios) which depends upon this package.
+
+### For testing purposes
+
+Reference in Package.swift:
+```
+...
+.package(url: "https://github.com/xmtp/xmtp-rust-swift", from: "0.2.0-beta0")
+...
+```
+Reference in Podspec:
+```
+...
+spec.dependency 'XMTPRust', '= 0.2.0-beta0'
+...
+```
 
 ## Diagram
 
 ```
 ┌────────────────────────────────────┬─────────────────────────────┬─────────────────────────────────┐
 │                                    │                             │                                 │
-│  xmtp/libxmtp: Shared Rust Code    │  xmtp/xmtp_rust_swift       │xmtp/xmtp-ios - iOS SDK          │
-│    - xmtp-networking               │  - Git repo to host Swift   │- Existing iOS xmtp SDK          │
-│    - xmtp-proto                    │  Package                    │- Add xmtp_rust_swift as Swift   │
-│    |                               │  - ApiService.swift is      │ package dep via Github url      │
-│    ---> bindingx/xmtp_rust_swift   │  thin wrapper of XMTPRust   │- Consumes xmtp/proto on branch  │
-│                                    │    ┌───────────────────┐    │   without gRPC Swift files      │
-│ ┌──────────┐     ┌───────────────┐ │ ┌─►│-Package.swift     │    │- Key: no more GRPC dep!         │
-│ │xmtp-proto├────►│xmtp-networking│ │ │  │-XMTPRust.xcframe..│    │    ┌──────────────────┐         │
-│ └──────────┘     └─────┬─────────┘ │ │  │-Sources/...       │    │    │                  │         │
-│                        │           │ │  │ -ApiService.swift │    │    │ Package.swift    │         │
-│        ┌───────────────▼────┐      │ │  └─────────┬─────────┘    │ ┌──►                  │         │
-│        │libxmtp/bindings/   │      │ │            │  Swift Pkg   │ │  │ - ApiClient.swift│         │
-│        │  xmtp_rust_swift   │      │ │            └──────────────┼─┘  │  - Uses XMTPRust │         │
+│  xmtp/libxmtp: Shared Rust Code    │  xmtp/xmtp-rust-swift       │ xmtp/xmtp-ios - iOS SDK         │
```

Comment:
Curious what you've been using to create these ASCII diagrams

Reply:
I use https://asciiflow.com/#/ then manually shift things around in Vim